assets/stylesheets/app.scss

```diff
@@ -4,15 +4,15 @@
 @import "mixins";
 
 // govuk-elements
-@import "govuk-elements/helpers";
-@import "govuk-elements/reset";
-@import "govuk-elements/elements-typography";
-@import "govuk-elements/forms";
-@import "govuk-elements/buttons";
-@import "govuk-elements/details";
-@import "govuk-elements/lists";
-@import "govuk-elements/panels";
-@import "govuk-elements/tables";
+@import "../../node_modules/govuk-elements-sass/public/sass/elements/helpers";
+@import "../../node_modules/govuk-elements-sass/public/sass/elements/reset";
+@import "../../node_modules/govuk-elements-sass/public/sass/elements/elements-typography";
+@import "../../node_modules/govuk-elements-sass/public/sass/elements/forms";
+@import "../../node_modules/govuk-elements-sass/public/sass/elements/buttons";
+@import "../../node_modules/govuk-elements-sass/public/sass/elements/details";
+@import "../../node_modules/govuk-elements-sass/public/sass/elements/lists";
+@import "../../node_modules/govuk-elements-sass/public/sass/elements/panels";
+@import "../../node_modules/govuk-elements-sass/public/sass/elements/tables";
```

Comment:
This path is likely to change depending on the installation. You'll need to do something similar to working out the module path for govuk_frontend_toolkit in https://github.com/UKHomeOffice/passports-frontend-toolkit/blob/master/assets/stylesheets/mixins.scss.tpl. Have a look at https://github.com/UKHomeOffice/passports-frontend-toolkit/blob/master/build